### PR TITLE
Setup LD options to include /usr/local/lib for libluajit in search path

### PR DESCRIPTION
**Pull request by Sauraus**
Signed-off-by: Antek S. Baranski <antek.baranski@gmail.com>

### CODE REVIEW

File: recipes/lua.rb [hunk 1]
```diff
@@ -34,11 +34,11 @@
     cd luajit-#{node['nginx']['luajit']['version']}/LuaJIT-#{node['nginx']['luajit']['version']}
     make && make install
     ldconfig
+    export LUAJIT_INC="/usr/local/include/luajit-2.0"
```
Comment (mitch-roblox) [inline recipes/lua.rb]: Why the exports here? Seems like they'd get set only for that execute block and then no longer be set? Maybe I'm missing something, but it seems like they'd still need to be part of the nginx_source_env hash used to set the `environment` attribute on the nginx compile block.

**Comment by mitch-roblox** [comment]
Not a change you made, but both the cookbook and recipe comment at the top of the file are wrong.

File: recipes/lua.rb (hunk 1 above)
Comment (Sauraus) [inline recipes/lua.rb]: Exploring another fix right now, hold on with the merge.

Comment (Sauraus) [inline recipes/lua.rb]: Ok fix is ready and verified to work.

**Pull request merged**